scripts/apply_trilinos_patches.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+hashes=('545269e1838282d98c4a113e17e6322d9c87fe8b')
+
+cd ${TRILINOS_DIR}
```

Comment:
Do the check if variable exists first, something like
```bash
if [ -z TRILINOS_DIR ]...
```
or `${TRILINOS_DIR:"Need to set TRILINOS_DIR"`